code4lib/2021.code4lib.org
# Update Homepage Event Cards for 2021

To Test:
- Switch conference mode to `peri`
- Confirm that homepage event cards are accurate and link to the proper events

This PR does not make any adjustments to the hero content.

Closes #182
